app/presenters/schools/courses/certificates_presenter.rb

```diff
@@ -26,6 +27,11 @@ def self.certificate_details(certificate)
 
       private
 
+      def can_be_auto_issued
+        top_level = @course.levels.order(number: :desc).first.id
+        @course.target_groups.where(level_id:top_level,milestone:true).exists?
```

Comment:
@a-c-sreedhar-reddy I've noticed in your PR-s that your Ruby code formatting is inconsistent.

Please use an auto-formatter of some sort. The easiest option is to install the [rufo](https://github.com/ruby-formatter/rufo) rubygem globally, and then configure that with the `Ruby` addon in VSCode.

<img width="693" alt="Screenshot 2020-10-07 at 11 43 09 AM" src="https://user-images.githubusercontent.com/98546/95294098-4c1abc00-0892-11eb-9223-54071a2aa370.png">